src/common/queries/payment-terms.ts

```diff
@@ -52,3 +54,15 @@ export function bulk(
     ids: id,
   });
 }
+
+export function useBlankPaymentTermQuery() {
+  return useQuery<PaymentTerm>(
+    '/api/v1/payment_terms/create',
+    () =>
+      request('GET', endpoint('/api/v1/payment_terms/create')).then(
+        (response: GenericSingleResourceResponse<PaymentTerm>) =>
+          response.data.data
+      ),
+    { staleTime: Infinity }
```

Comment:
This also needs query-level permissions. Do we want to enable this query only if admin? @turbo124

Reply:
@beganovich yes, admin only

Reply:
Condition made!